spaghetto/flake.nix

```diff
@@ -0,0 +1,84 @@
+{
+  description = "PureScript app flake";
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/ec322bf9e598a510995e7540f17af57ee0c8d5b9";
+
+    flake-utils.url = "github:numtide/flake-utils";
+
+    flake-compat = {
+      url = "github:edolstra/flake-compat";
+      flake = false;
+    };
+
+    # TODO switch to thomashoneyman/purescript-overlay after https://github.com/thomashoneyman/purescript-overlay/pull/40 is merged
+    purescript-overlay = {
+      url = "github:deemp/purescript-overlay";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+
+    easy-purescript-nix = {
+      url = "github:justinwoo/easy-purescript-nix";
+      inputs.flake-utils.follows = "flake-utils";
+    };
```

Comment:
Presumably this _and_ purescript-overlay are both included because `purifix` depends on easy-purescript-nix. Maybe if the easy-purescript-nix dependency is forced then it should just be used in place of purescript-overlay.